### DISCUSSION TITLE
Anybody's Z2M Add A Device Even When Join Is Disabled?

### DISCUSSION THREAD

**Discussion by danieledwardgeorgehitchcock** [Koenkk/zigbee2mqtt#9456] [General]
I have found that if a device has been previously added to my Z2M network, and then removed, I do not need Z2M to be in Join mode for the device to be detected when it is in pair mode, and re-joins my network.

I have only managed to do this when the device has previously been joined to my network. New devices don't exhibit the same behaviour (maybe they do, but I haven't seen it yet).

Using latest Z2M version with latest ZZH stick and firmware.

**Comment by RobLeighton67** [comment]
I think I may have had this situation - my suspicions pointed to the Coordinator backup that seemed to contain any previously joined device, not just the currently 'present' devices ?
